Core/HLE/sceFont.cpp

```diff
@@ -645,9 +645,26 @@ static void __LoadInternalFonts() {
 	if (!pspFileSystem.GetFileInfo(fontPath).exists) {
 		pspFileSystem.MkDir(fontPath);
 	}
+	if (pspFileSystem.GetFileInfo("disc0:/PSP_GAME/USRDIR/zh_gb.pgf").exists) {
+		for (size_t i = 0; i < ARRAY_SIZE(fontRegistry); i++) {
```

Comment:
This is orignal log:
https://gist.github.com/sum2012/8357a3503e4374e862221e097c72fd36
It seem that the chinese team patched the font.prx (or use scereg ?) to force to use it's custom font.
JPCSP also have this problem although it has scereg.